lib/wallet.js

```diff
@@ -315,6 +315,74 @@ class WalletClient extends Client {
     return this.get(`/wallet/${id}/coin/${hash}/${index}`);
   }
 
+  /**
+   * Get NameState from wallet
+   * @param {String} id
+   * @param {String} name
+   * @returns {Promise}
+   */
+
+  getName(id, name) {
+    return this.get(`/wallet/${id}/name`, { name });
+  }
+
+  /**
+   * Get all names wallet is managing
```

Comment:
Same as above. Perhaps: "Get name info for all names the specified wallet is managing.